docs/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Entries inside each section should be ordered by type:
 ## Python API
 
 ## CLI
+* [Added] `create_user()`, `delete_user()`, `set_role()` in `quilt3.admin` ([#3764](https://github.com/quiltdata/quilt/pull/3764))
```

Comment:
I don't mentioned removed functions because they were not documented and don't work for quite a long.